packages/venia-ui/lib/components/Logo/logo.js

```diff
@@ -18,14 +20,23 @@ const Logo = props => {
     const { height, width } = props;
     const classes = mergeClasses({}, props.classes);
 
+    const talonProps = useLogo({
+        query: GET_LOGO_DATA,
+        veniaLogo: { logo },
+        veniaWidth: { width },
```

Comment:
Same here, let's call it `defaultWidth` and not wrap:
```suggestion
        defaultWidth: width,
```

Reply:
Added commit with your suggestion.